CMakeLists.txt

```
@@ -306,13 +306,18 @@ endif()
 
 find_package(Perl)
 
-option(BUILD_DOCS "to build manual pages" ON)
-option(ENABLE_MANUAL "to provide the built-in manual" OFF)
-
-if(ENABLE_MANUAL AND PERL_FOUND)
-  curl_nroff_check()
-  if(NROFF_USEFUL)
-    set(USE_MANUAL ON)
+option(BUILD_DOCS "to build libcurl man pages" ON)
+option(ENABLE_MANUAL "to provide the built-in manual and man page for curl tool" OFF)
+
+if(ENABLE_MANUAL OR BUILD_DOCS)
+  if(PERL_FOUND)
+    curl_nroff_check()
+    if(NROFF_USEFUL)
+      set(HAVE_MANUAL_TOOLS ON)
+    endif()
+  endif()
+  if(NOT HAVE_MANUAL_TOOLS)
+    message(WARNING "Perl not found, or nroff not useful. Will not provide built-in manual.")
```

Since this stops *any* documentation from being built, I'd say that.  So:

```suggestion
    message(WARNING "Perl not found, or nroff not useful. Will not build manuals.")
```